src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -884,7 +881,7 @@ export class BaseEditCourseForm extends React.Component {
               disabled={disabled}
               optional
             />
-            {displaySkills
+            {skillNames?.length > 0
```

Comment:
do we want to add `skillNames` not null check here i.e. `skillNames && skillNames.length > 0` is that is not necessary here?

Reply:
The [Optional Chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) operator (?) will take care of that.